netci.groovy

```diff
@@ -831,9 +835,18 @@ combinedScenarios.each { scenario ->
                                         // Get corefx
                                         buildCommands += "git clone https://github.com/dotnet/corefx fx"
                                         
+                                        // Set environment variable
+                                        def setEnvVar = ''
+                                        def envVars = Constants.jitStressModeScenarios[scenario]
+                                        envVars.each{ VarName, Value   ->
+                                            if (VarName != '') {
+                                                setEnvVar += " ${VarName}=${Value}"
```

Comment:
I usually set environment variable w/o ';' but I usually set only one variable.. I tried the following and it seems to work without ';'. Let me know if my understanding is not correct.


```
$ cat t.sh
#!/bin/sh
echo $VX $VY
$ VX=1 VY=2 ./t.sh
1 2
$ VX=1; VY=2; ./t.sh
<no output>
$ export VX=1; export VY=2; ./t.sh
1 2
$
```